confignode/src/main/java/org/apache/iotdb/confignode/manager/load/heartbeat/DataNodeHeartbeatCache.java

```diff
@@ -112,12 +95,6 @@ public NodeStatus getNodeStatus() {
     }
 
     // Return a copy of status
-    switch (status) {
-      case Running:
-        return NodeStatus.Running;
-      case Unknown:
-      default:
-        return NodeStatus.Unknown;
-    }
+    return NodeStatus.valueOf(status.getStatus());
```

Comment:
same impl to the `getNodeStatus` of `ConfigNodeHeartbeatCache`.

![image](https://user-images.githubusercontent.com/6756545/186399558-6d9ea3ff-e7f7-40dc-b553-c07adcee7058.png)

Reply:
Fixed~